recipe/0001-setup.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/setup.py b/setup.py
+index 985eb49..5e313ba 100644
+--- a/setup.py
++++ b/setup.py
+@@ -22,7 +22,7 @@ requirements = [
+     "pandas>=1.5.0, <2.0.0",
+     "patsy>=0.5.1",
+     "plotly>=4.12.0",
+-    "pmdarima>=1.8.0, <=1.8.5",
++    "pmdarima>=1.8.0",
+     "pytest>=4.6.5",
+     "pytest-runner>=5.1",
```

Comment:
```suggestion
#     "pytest>=4.6.5",
#     "pytest-runner>=5.1",
```

Reply:
both they should be removed from the requirements, otherwise users without `pytest` installed will see their `pip check` failing (ours does not fail because we do have `pytest` in the `test` section).

Reply:
Done